apps/frontend/src/app/5_pages/ErrorPage/ErrorPage.tsx

```diff
@@ -1,16 +1,26 @@
 import React, { FC } from 'react';
 
 import { t } from 'i18next';
+import { Helmet } from 'react-helmet-async';
 import { useRouteError } from 'react-router-dom';
 
 import { Footer, Header } from '../../3_organisms';
 import { translations } from '../../../locales/i18n';
+import { NotFoundPage } from '../NotFoundPage/NotFoundPage';
 
 export const ErrorPage: FC = () => {
-  const error = useRouteError() as Error & { statusText?: string };
+  const error = useRouteError() as Error & {
```

Comment:
```suggestion
  const error = useRouteError() as ErrorRouteProps
```
Define a new type (e.g. `ErrorRouteProps`) outside the component and just use it here.